teps/0112-replace-volumes-with-workspaces.md

```diff
@@ -256,25 +268,186 @@ spec:
 
 Pros:
 - Flexible, not tied to Kubernetes
+- Workspace "types" could be leveraged in the future to label workspaces as "input" and "output", to address
+  [workspace dependencies](./0063-workspace-dependencies.md). (This is also a potential downside, as it signals
+  that "type" is too generic.)
 
 Cons:
 - `Task` authors may want to use other types of Volumes as part of Task implementation, such as hostPath volumes
 
-### Support for "dynamic" Workspaces
+#### Allow running all Steps in the same container
```

Comment:
I don't understand what this has to do with volumes and workspaces ? What does it solves ?

Reply:
I updated the wording a bit to make this more clear. FWIW I don't really think this is the right way forward. I put this in because I was exploring other CI/CD systems and whether they have a solution for this, and noticed that they don't because Tekton is the only one as far as I can tell that runs Step-like things in separate containers, and thought the connection was worth noting.

Reply:
Dagger does as well (per container "steps").. with a little twist, it can run a step in a container generated by a previous step (something that cannot be done for Tekton)… (but steps and task are the same thing in dagger).